File: Python PowerPoint Examples/DocumentApiDemo/convertPpt.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.document_api.DocumentApi(configuration)

name = "ConvertPpt.pptx"
password = ""
storage = ""
format = "Pptx"  # Supported formats:Pptx/Ppt/Pps/Ppsx/Odp/Pdf/Xps/Ps/Pcl
folder = "input"

result = api.convert_ppt(name, format=format, password=password, storage=storage, folder=folder)

File: PHP PowerPoint Examples/DocumentApiDemo/convertPpt.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\DocumentApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new DocumentApi($configuration);

$name = "ConvertPpt.pptx";
$format = "Pptx"; //Available format: Pptx, Ppsx, Ppt, Pps, Odp, Pdf, Xps, Ps, Pcl
$options = null;
$password = null;
$folder = "Powerpoint";
$storage = null;
$result = $apiInstance->convertPpt($name, $format, $options, $password, $storage, $folder);
print_r($result);